packages/react-query-devtools/package.json

```diff
@@ -10,36 +10,46 @@
     "type": "github",
     "url": "https://github.com/sponsors/tannerlinsley"
   },
-  "module": "build/esm/index.js",
-  "main": "build/cjs/packages/react-query-devtools/src/index.js",
-  "browser": "build/umd/index.production.js",
-  "types": "build/types/react-query-devtools/src/index.d.ts",
-  "files": [
-    "build/*",
-    "src"
-  ],
+  "types": "build/lib/index.d.ts",
+  "main": "build/lib/index.js",
   "exports": {
-    "development": {
-      "require": "./build/cjs/packages/react-query-devtools/src/index.js",
-      "default": "./build/esm/index.js"
+    ".": {
+      "types": "./build/lib/index.d.ts",
+      "import": "./build/lib/index.mjs",
+      "default": "./build/lib/index.js"
+    },
+    "./production": {
+      "types": "./build/lib/index.d.ts",
```

Comment:
These exports won't work with TypeScript versions < 4.7 or bundlers that don't support `exports` (older bundlers and React Native). Not sure what platforms/versions we're targeting but something to keep in mind.

If we do want to support them, we'll need to match the top level file structure of the package with the entry points in `exports`.

Reply:
yep, we are aware of that. I've documented that here: Do you think it's good enough?

https://github.com/TanStack/query/blob/beta/docs/devtools.md#modern-bundlers

Reply:
Ah yeah that's good